torch/cuda/memory.py

```diff
@@ -994,9 +996,13 @@ def id(self) -> Tuple[int, int]:
 
     @property
     def allocator(self) -> Optional[_cuda_CUDAAllocator]:
-        r"""Returns the allocator this MemPool routes allocations to"""
+        r"""Returns the allocator this MemPool routes allocations to."""
         return super().allocator
 
+    def use_count(self) -> int:
+        r"""Returns the reference count of this pool."""
+        return super().use_count()
```

Comment:
You probably don't actually need this anymore right? Probably harmless enough though

Reply:
Yes. I use this in the unit-tests and this should be useful for debugging purposes. For instance, a user can query use_count to see if they have deleted all the tensors in the pool (before deleting the pool and issuing a `torch.cuda.empty_cache()`).